src/docs/development/packages-and-plugins/androidx-compatibility.md

```diff
@@ -4,18 +4,17 @@ description: How to fix AndroidX incompatibilities that have been detected by th
 ---
 
 {{site.alert.note}}
-  You might be directed to this page if the framework detects a problem in your
-  Flutter app involving AndroidX incompatibilities.
+  You might be directed to this page if the framework detects a
+  problem in your Flutter app involving AndroidX incompatibilities.
 {{site.alert.end}}
 
-Android code often uses the
-[`android.support`]({{site.android-dev}}/topic/libraries/support-library/)
-libraries to ensure backwards compatibility. The `android.support`
-libraries are deprecated, and replaced with
-[AndroidX]({{site.android-dev}}/jetpack/androidx/).
+Android code often uses the [`android.support`][]
```

Comment:
There are a few instances of replacing links with `[foo][]`.  Why is this preferable?  It seems like it's easier to end up with broken links.

Reply:
It is our convention. The text is far more readable without the inline link. And Travis tells us of broken links.

Reply:
Is Travis a human or a bot?  It would nice to have the validation automated since it's almost impossible to review visually.

Reply:
Travis is a CI tool. We used to use it on flutter/flutter, but now it's Cirrus. Very much a bot.  :) . We also run a weekly linkcheck over the site.

Reply:
Okay, phew.  I didn't want a real Travis to be hurt about me not knowing if they were a bot or not.  I think bot Travis will get over it.